src/main/java/dk/cngroup/lentils/service/convertors/ObjectMapper.java

```diff
@@ -1,6 +1,12 @@
 package dk.cngroup.lentils.service.convertors;
 
+import dk.cngroup.lentils.dto.CypherFormDTO;
+import dk.cngroup.lentils.entity.Cypher;
+
 public interface ObjectMapper {
     void map(Object source, Object destination);
     <T> T map(Object source, Class<T> destinationClassName);
+    Cypher map(CypherFormDTO source);
```

Comment:
Všechno dobrý až do tohohle bodu. Nevím, jestli se mě líbí tyhle specializace v tomhle interfacu.

Za chvíli budeš mít hromadu takových konkrétních metod v jedné třídě a stane se z ní jedna obrovská. Trochu jako God Object. Jako by si spojil všechny konvertery a třída bude mít hromadu nesouvisejících zodpovědností.

To se raději přikláním dodatečné atributy namapovat v kontroleru nebo někde jinde.

Nebo jak se to zdá @mjanousek ?

Reply:
Urcite se nesmi pridavat specificke metody. Proc na to mevyuzijes to metodu `<T> T map(Object source, Class<T> destinationClassName);`?

Reply:
přesunul jsem teda mapování organizátorů šifry do kontroleru a smazal ty specializace :)